Documentation/project-docs/performance-tests.md

```diff
@@ -44,7 +44,10 @@ For the time being, perf tests should reside within their own "Performance" fold
 Start by adding the following lines to the tests csproj:
 ```
  <!-- Performance Tests -->  
-  <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true'">  
+  <PropertyGroup Condition="'$(Performance)' == 'true'">
+    <IncludePerformanceTests>true</IncludePerformanceTests>
+  </PropertyGroup>
```

Comment:
The purpose of it is so that IncludePerformanceTests is only set to true for csprojs that actually contain perf tests. When Performance=true is passed to a non-perf project, no actual perf tests are run since they depend on IncludePerformanceTests being set which is missing from normal test projects.

I couldn't think of a better way of distinguishing between test projects that include perf tests and test projects that do not. If I could do that in dir.props it would definitely be preferable though; any ideas?